pages/issue/IssueHandler.inc.php

```diff
@@ -147,16 +146,16 @@ public function archive($args, $request)
         $count = $context->getData('itemsPerPage') ? $context->getData('itemsPerPage') : Config::getVar('interface', 'items_per_page');
         $offset = $page > 1 ? ($page - 1) * $count : 0;
 
-        $params = [
-            'contextId' => $context->getId(),
-            'orderBy' => 'seq',
-            'orderDirection' => 'ASC',
-            'count' => $count,
-            'offset' => $offset,
-            'isPublished' => true,
-        ];
-        $issues = iterator_to_array(Services::get('issue')->getMany($params));
-        $total = Services::get('issue')->getMax($params);
+        $collector = Repo::issue()->getCollector()
+            ->limit($count)
+            ->offset($offset)
+            ->filterByContext($context->getId())
+            ->orderBy(Collector::ORDERBY_SEQUENCE)
+            ->orderDirection(Collector::ORDER_DIR_ASC)
+            ->filterByPublished(true);
+
+        $issues = Repo::issue()->getMany($collector)->toArray();
+        $total = Repo::issue()->getCount($collector);
```

Comment:
You'll need to rmeove the limit and offset before calling getCount. I often do this:

```
$collector->filterByX(...);
$total = ...->getCount($collector);
$issues = ...->getMany($collector->limit(...)->offset(...);
```